ui/templates/index.html

```diff
@@ -5,5 +5,6 @@
 
 {% block content %}
 <h1>{% trans "Hi, I'm micromasters" %}</h1>
+<div id="container"></div>
```

Comment:
I don't think we're using React at all for this page since it's not part of the dashboard